doc/classes/ResourceLoader.xml

```diff
@@ -76,7 +76,7 @@
 				The registered [ResourceFormatLoader]s are queried sequentially to find the first one which can handle the file's extension, and then attempt loading. If loading fails, the remaining ResourceFormatLoaders are also attempted.
 				An optional [param type_hint] can be used to further specify the [Resource] type that should be handled by the [ResourceFormatLoader]. Anything that inherits from [Resource] can be used as a type hint, for example [Image].
 				The [param cache_mode] property defines whether and how the cache should be used or updated when loading the resource. See [enum CacheMode] for details.
-				Returns an empty resource if no [ResourceFormatLoader] could handle the file.
+				Returns an empty resource if no [ResourceFormatLoader] could handle the file, and logs an error if no file is found at the specified path.
```

Comment:
```suggestion
				Returns an empty resource if no [ResourceFormatLoader] could handle the file, and prints an error if no file is found at the specified path.
```

Reply:
I amended the wording change to my initial commit  👍